src/theory/smt_engine_subsolver.cpp

```diff
@@ -126,7 +126,7 @@ Result checkWithSubsolver(Node query,
   initializeSubsolver(smte, opts, logicInfo, needsTimeout, timeout);
   smte->assertFormula(query);
   r = smte->checkSat();
-  if (r.getStatus() == Result::SAT)
+  if (r.getStatus() == Result::SAT || r.getStatus() == Result::UNKNOWN)
```

Comment:
What are gonna be the guarantees here for `getValue` when we have an `UNKNOWN` result? Can't this lead to issues?

Reply:
The caller can sort this out by checking for the result.